README.md

```diff
@@ -1,12 +1,16 @@
 # cms-l1t-analysis
 Software package to analyse L1TNtuples
 
+ - [![Build Status](https://travis-ci.org/cms-l1t-offline/cms-l1t-analysis.svg?branch=master)](https://travis-ci.org/cms-l1t-offline/cms-l1t-analysis)
+ - [![DOI](https://zenodo.org/badge/80877637.svg)](https://zenodo.org/badge/latestdoi/80877637)
+
 
 ## DEV
 ```bash
 git clone https://github.com/<your github user name>/cms-l1t-analysis.git
 cd cms-l1t-analysis
 git remote add upstream https://github.com/cms-l1t-offline/cms-l1t-analysis.git
+# only on non Scientific Linux machines (e.g. OS X, Windows, Ubuntu, etc)
 vagrant up
 vagrant ssh
 cd /vagrant
```

Comment:
Do you think mention which is the last command that only applies to non-SL machines?  I think everything below this line everyone runs right?